charts/index.yaml

```diff
@@ -1,28 +1,28 @@
 apiVersion: v1
 entries:
-  federation-v2:
+  kubefed:
```

Comment:
I don't think any of the previous releases should be renamed. They should remain in the index as federation-v2.

Reply:
Got it.